From 39ef2b9abccba9798ab5daa19d4382660d1d5fb2 Mon Sep 17 00:00:00 2001
From: Vicentiu Galanopulo <vicentiu@balena.io>
Date: Fri, 24 Apr 2020 14:01:54 +0200
Subject: [PATCH] Remove scanning of USB as boot devices at startup

When u-boot looks for bootable devices it scans all the
etcher-pro board connected USB drives. This slows down
the boot process significantly

Upstream-Status: Inappropriate [configuration]
Signed-off-by: Vicentiu Galanopulo  <vicentiu@balena.io>
---
 include/configs/cl-som-imx8.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/cl-som-imx8.h b/include/configs/cl-som-imx8.h
index 862a67c08a..8aa4704c2c 100644
--- a/include/configs/cl-som-imx8.h
+++ b/include/configs/cl-som-imx8.h
@@ -133,7 +133,7 @@
 	"run resin_set_kernel_root; run set_os_cmdline; " \
 	"setenv mmcdev ${resin_dev_index};" \
 	"setenv mmcbootpart ${resin_boot_part};" \
-	"setenv mmcdev 1; run mmc_boot; run usb_boot; setenv mmcdev 0; run mmc_boot;"
+	"setenv mmcdev 1; run mmc_boot; setenv mmcdev 0; run mmc_boot;"
 
 /* Link Definitions */
 #define CONFIG_LOADADDR			0x40480000
-- 
2.17.1

